src/Components/Artwork/Details/Details.tsx

```diff
@@ -430,12 +403,6 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
           registrationEndsAt
           onlineBiddingExtended
         }
-        partnerOffer {
```

Comment:
Hooray!

I wonder if `primaryLabel` needs to also be removed here (or updated in MP), as I _think_ it can return the personalized 'offer' label. Or is this perhaps what you refer to in the PR description about the precedence of only displaying one label at most?

Reply:
Yeah, I decided to [remove partner offer from the primary label field](https://github.com/artsy/force/pull/14807/commits/8b7c310770df40cff92bc7e6773471ed9280ca4b) for that reason. After https://github.com/artsy/metaphysics/pull/6220, the primary label will be fully cacheable. I think it's also more aligned with the pattern that separates authenticated data out from non-auth data.

I shared more context on [Slack](https://artsy.slack.com/archives/C02JHHHKP5K/p1731340451105599).

Reply:
Cool - makes sense. I think you're right that schema-wise - of course it's a case-by-case basis - but not mixing authenticated and cacheable data as part of the resolving of the same _field_ (in this case `collectorSignals` under the `Artwork` type)  - is something we should be doing.

I think collector signals _might_ be the only case of that really (🤞), as typically an entire _field_ would be either authenticated or not - and not a more generic-y mix of both as collector signals is.